src/dbus_api/api.rs

```diff
@@ -282,9 +282,31 @@ pub fn connect(engine: Rc<RefCell<Engine>>)
 
     c.register_name(STRATIS_BASE_SERVICE, NameFlag::ReplaceExisting as u32)?;
 
+    process_deferred_actions(&c, &mut tree, &dbus_context)?;
+
     Ok((c, tree, dbus_context))
 }
 
+fn process_deferred_actions(c: &Connection,
```

Comment:
Are you able to pass only the actions, not the whole context.?

Reply:
I tried, but after spending way too much time I gave up.